packages/nimble-blazor/Tests/NimbleBlazor.Tests.Acceptance/Pages/ThemeProvider.razor

```diff
@@ -0,0 +1,55 @@
+﻿@page "/ThemeProvider"
+@namespace NimbleBlazor.Tests.Acceptance.Pages
+@inherits LayoutComponentBase
+
+<NimbleButton @onclick="SetInvalidLang">Set Invalid Lang</NimbleButton>
+<NimbleButton @onclick="SetValidLang">Set Valid Lang</NimbleButton>
+
+<NimbleThemeProvider @ref="_themeProvider" Lang="@Lang"></NimbleThemeProvider>
+
+<NimbleCheckbox value="@IsValid">IsValid</NimbleCheckbox>
+<NimbleCheckbox value="@InvalidLang">InvalidLang</NimbleCheckbox>
+<input value="@Lang" />
+<input value="@(IsValid ? "valid" : "invalid")" />
```

Comment:
Are these two `input` element necessary? They don't appear to be used by the test.

Reply:
I had intended to remove them when I was done debugging/fixing the test, but I forgot.